utils/constants/networking.go

```diff
@@ -30,7 +30,8 @@ const (
 	// MinConnectedStakeBuffer is the safety buffer for calculation of MinConnectedStake.
 	// This increases the required stake percentage above alpha/k. Must be [0-1]
 	// 0 means MinConnectedStake = alpha/k, 1 means MinConnectedStake = 1 (fully connected)
-	MinConnectedStakeBuffer = .2
+	MinConnectedStakeBuffer  = .2
+	DefaultMinConnectedStake = .8
```

Comment:
Based on the above comment, I don't think this const will be needed anymore.

Reply:
removed